packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -2341,12 +2340,12 @@ function pushStyle(
       } else {
         resource.props.hrefs.push(href);
       }
-      resumableState.stylesMap.set(key, resource);
-      if (renderState.boundaryResources) {
-        renderState.boundaryResources.add(resource);
-      }
+      resumableState.stylesMap[key] = null;
       pushStyleContents(resource.chunks, props);
     }
+    if (renderState.boundaryResources) {
+      renderState.boundaryResources.add(resource); // TODO
+    }
```

Comment:
@gnoff I moved this out because it appears to be inconsistent with `stylesheet`. Either we have to add it to boundary resources each time it's discovered or maybe not all.

If we don't need this to be coordinated because it's always sync available and will always flush early, then maybe we never add it to boundaryResources but if we need it to be emitted, then it seems like we must always add it.

Reply:
Yeah I think this was just my mistake. however, we should reconcider whether it makes sense to hold back some style tags when completing boundaries vs just flushing everything we've accumulated so far. If we always flush all style precedences on each tick then we don't need to track